drivers/usbc/vbus/Kconfig.vbus_adc

```diff
@@ -0,0 +1,12 @@
+# VBUS device configuration options
+
+# Copyright 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_VBUS_ADC := vbus-adc
```

Comment:
Please take a look at support for "devicetree Kconfig symbol" introduced by https://github.com/zephyrproject-rtos/zephyr/pull/47196

Reply:
Thanks. I've updated the Kconfig